addons/zebra-striping/addon.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "Alternating nested block colors",
+  "description": "Makes blocks of the same category alternate between lighter and darker shades when nested inside each other. This is also known as zebra striping.",
```

Comment:
> This is also known as zebra striping.

Is "this is" neccessary? Because I feel in a short description, it sounds better to leave that out, it's obvious what does it refer to. 
((For real, this is a very minor thing, just wanted to ask in the purpose of learning and hearing other opinions.))

Reply:
It could be considered unnecessary, but I don't think it matters much.